### PR TITLE
[master < T600] Implement << operator for mgp::Value

### PR DESCRIPTION
**Pull request by imilinovic**
[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [X] Write a release note here, including added/changed clauses
You can use << operator to print mgp::Value for all types except mgp::Path, mgp::List and mgp::Map.
- [ ] Tag someone from docs team in the comments

### CODE REVIEW

File: include/mgp.hpp [hunk 1]
```diff
@@ -3480,6 +3482,54 @@ inline bool Value::operator<(const Value &other) const {
...
+    case mgp::Type::Int:
+      return os << std::to_string(value.ValueInt());
+    case mgp::Type::Double:
+      return os << std::to_string(value.ValueDouble());
+    case mgp::Type::String:
+      return os << std::string(value.ValueString());
+    case mgp::Type::List:
+      return os << "List[]";
+    case mgp::Type::Map:
+      return os << "Map[]";
```
Comment (antoniofilipovic) [inline include/mgp.hpp:3502]: Do we want to throw an exception maybe for this? As it is not implemented?

File: include/mgp.hpp [hunk 2]
```diff
@@ -3480,6 +3482,54 @@ inline bool Value::operator<(const Value &other) const {
...
+    case mgp::Type::Node:
+      return os << "Node[" + std::to_string(value.ValueNode().Id().AsInt()) + "]";
+    case mgp::Type::Relationship:
+      return os << "Relationship[" + std::to_string(value.ValueRelationship().Id().AsInt()) + "]";
+    case mgp::Type::Path:
+      return os << "Path[]";
```
Comment (antoniofilipovic) [inline include/mgp.hpp:3508]: And for this too to throw an exception?

**Review by antoniofilipovic — APPROVED** [review]
Looks good

**Comment by antoniofilipovic** [comment]
@vpavicic release note is above

**Comment by vpavicic** [comment]
is this changing just the cpp API? (some PRs change both the cpp and c)

**Comment by imilinovic** [comment]
Only cpp API.

**Pull request merged**